pontoon/base/static/js/translate.js

```diff
@@ -1566,6 +1566,19 @@ var Pontoon = (function (my) {
 
 
     /*
+     * Prevent too long translation submissions
+     */
+    translationLengthLimitExceeded: function (translation) {
+      if (this.translationLengthLimit && this.stripHTML(translation).length > this.translationLengthLimit) {
+        this.endLoader('Translation too long.', 'error');
```

Comment:
But I see what you did here -> You want to avoid duplication, but still leaving side-effect in a method that should return a simple boolean, can be a little misleading.  Anyway, as there's no clear solution - I trust your judgment.